node/src/components/consensus/protocols/highway.rs

```diff
@@ -41,7 +41,11 @@ use self::{round_success_meter::RoundSuccessMeter, synchronizer::Synchronizer};
 
 /// Never allow more than this many units in a piece of evidence for conflicting endorsements,
 /// even if eras are longer than this.
-const MAX_ENDORSEMENT_EVIDENCE_LIMIT: u64 = 10000;
+const MAX_ENDORSEMENT_EVIDENCE_LIMIT: u64 = 10_000;
+/// The number of milliseconds after which an alert is raised if no progress was made.
+const STANDSTILL_ALERT_MILLIS: u64 = 5 * 60_000;
```

Comment:
So it will shut down the node if it hasn't changed an era in 5 minutes?

Reply:
Yes… but it will only stop changing the era itself once it goes into pause mode, which is three maximum-length eras after it notices that not enough weight is online anymore.